packages/sit-onyx/src/components/OnyxTooltip/OnyxTooltip.vue

```diff
@@ -0,0 +1,115 @@
+<script lang="ts" setup>
+import { createTooltip } from "@sit-onyx/headless";
+import OnyxIcon from "../OnyxIcon/OnyxIcon.vue";
+import type { OnyxTooltipProps } from "./types";
+
+const props = withDefaults(defineProps<OnyxTooltipProps>(), {
+  color: "neutral",
+  position: "top",
+  fitParent: false,
+});
+
+defineSlots<{
+  /**
+   * Default slot where the parent content is placed that should open/close the tooltip.
+   */
+  default(props: Record<string, unknown>): unknown;
+}>();
+
+const {
+  elements: { trigger, tooltip },
+} = createTooltip({});
+</script>
+
+<template>
+  <div class="onyx-tooltip-wrapper">
+    <div
+      v-bind="tooltip"
+      class="onyx-tooltip onyx-text--small onyx-truncation-multiline"
```

Comment:
I can not find the implementation for this class anywhere, where does it come from?

also, wasn't this feature removed from the ticket?
```suggestion
      class="onyx-tooltip onyx-text--small"
```

Reply:
Its in the `styles/fonts.scss` file :) No truncation is still part of the ticket, only the wedge alignment is out of scope

Reply:
ah right first I thought this means "truncation even if it is multiline".. sry ^^